scripts/outer/kubectl-kubestellar-prep_for_syncer

```diff
@@ -67,6 +68,8 @@ while (( $# > 0 )); do
 	    fi;;
 	(-s|--silent)
 	    silent="true";;
+	(-c|--central)
+	    central="true";;
```

Comment:
Please keep the documentation in `commands.md`, and printed in response to `-h`, consistent with the code.

Reply:
Removed this flag.

Reply:
done